controllers/util/secrets_util.go

```diff
@@ -230,7 +281,28 @@ func newS3ConfigurationSecret(s3SecretRef corev1.SecretReference, targetns strin
 	}
 }
 
-func newConfigurationPolicy(name string, object runtime.RawExtension) *cpcv1.ConfigurationPolicy {
+func newVeleroSecret(s3SecretRef corev1.SecretReference, fromNS, veleroNS, keyName string) *localSecret {
+	return &localSecret{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "Secret",
+			APIVersion: "v1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "v" + s3SecretRef.Name, // Disambiguate with a "v" in case fromNS and veleroNS are the same
```

Comment:
This should use the constant, but inventing a new name we control will make this easier to work with.

Reply:
This is done